.github/workflows/update-deps.yml

```diff
@@ -0,0 +1,63 @@
+name: Tools and deps update
+on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: 5 0 * * 0
```

Comment:
Do we need a schedule?

Reply:
It should avoid us forgetting to run it manually.